### ISSUE TITLE
Atlantis for iOS stopped working after upgrading to Mac OS Big Sur

### ISSUE THREAD

**Issue by axnmax** [ProxymanApp/atlantis#76]
Hi Team,

After I upgraded my Mac to Big Sur, my Atlantis is not able to track the network traffic in Proxyman. Can anyone help to resolve this issue?

Thanks in advnce.

**Comment by NghiaTranUIT** [comment]
Hey @axnmax, I'm using Big Sur and I would like to clarify that Atlantis works fine here 👍 

To investigate, please help me:
1. Make sure you turn OFF the Firewall in System Preference -> Security & Privacy -> Firewall
2. Please update to the latest Atlantis with `pod update`

and use the following code to connect to your macOS

```swift
Atlantis.start(hostName: "_your_host_name")
```
You can find the hostname in the Atlantis window from Proxyman for macOS 👍 

![atlantis_hostname](https://user-images.githubusercontent.com/5878421/121375176-bbd5cf00-c96a-11eb-8d7d-2d33399d4e94.png)

**Comment by axnmax** [comment]
Hi @NghiaTranUIT,

I followed the steps as described by you. 
Still it does not work for me. It used to work perfectly for me before I upgraded to Big Sur.

**Comment by NghiaTranUIT** [comment]
I just tested again and it works with Atlantis 1.10.0 and Big Sur. I'm not sure why Atlantis doesn't work in your end 🤔 

When you open Atlantis, do you see any log message like:
```
---------------------------------------------------------------------------------
---------- 🧊 Atlantis is running (version 1.10.0)
---------- Github: https://github.com/ProxymanApp/atlantis
---------------------------------------------------------------------------------
[Atlantis] Looking for Proxyman app in the network...
Reachability Status Changed: reachable(Alamofire.NetworkReachabilityManager.NetworkReachabilityStatus.ConnectionType.ethernetOrWiFi)
[Atlantis] Found Proxyman with HostName = Nghias-Mac-mini.local.
[Atlantis] ✅ Connect to Nghias-Mac-mini.local.
```

If Bonjour Service finds any Macbook which has Proxyman, it will list it 
```
[Atlantis] Found Proxyman with HostName = Nghias-Mac-mini.local.
```

If you could not see anything, look like your network is blocking the Bonjour Service @axnmax

**Comment by axnmax** [comment]
Yes I am able to see the logs as you mentioned. But the Proxyman does not show any traffic.

---------------------------------------------------------------------------------
---------- 🧊 Atlantis is running (version 1.10.0)
---------- Github: https://github.com/ProxymanApp/atlantis
---------------------------------------------------------------------------------
[Atlantis] Looking for Proxyman app in the network...

[Atlantis] Found Proxyman with HostName = axnmax.local.
[Atlantis] ✅ Connect to axnmax.local.

**Comment by NghiaTranUIT** [comment]
Look like it's a bug @axnmax 🤔 

Atlantis is connecting to Proxyman, but it doesn't send any message.

If you don't mind, please help me to debug:
1. Start Proxyman and Atlantis => Make sure you can see the log: `[Atlantis] ✅ Connect to axnmax.local.`
2. Set a breakpoint on two place:

- https://github.com/ProxymanApp/atlantis/blob/9d125e08268437e4231bcbae99cccb337d5ee5df/Sources/Transporter.swift#L109
- https://github.com/ProxymanApp/atlantis/blob/9d125e08268437e4231bcbae99cccb337d5ee5df/Sources/Transporter.swift#L114
3. Try to make a request and see which breakpoint is stopped.
4. And set a breakpoint after Atlantis sends a message at
https://github.com/ProxymanApp/atlantis/blob/9d125e08268437e4231bcbae99cccb337d5ee5df/Sources/Transporter.swift#L137

Since I could not access to your codebase, so I could debug. Please help me if you don't mind. I appreciate it 🙇

**Comment by axnmax** [comment]
The control only reaches line 109. It did not go to lone 114 and 137 anytime.

**Comment by NghiaTranUIT** [comment]
Thanks for getting back to me. Just wondering: 

1. Which iOS version are you using?
2. Are you using Atlantis on a real iOS Device or iOS Simulator? 🤔

**Comment by axnmax** [comment]
1. iOS 13.2.3
2. Real iOS Device

Also there is an update on the breakpoints, last time it did not connect to the local host.  The control only reaches line 114. It did not go to line 109 and 137 anytime.

**Comment by NghiaTranUIT** [comment]
Thanks. I will investigate the bug. Meanwhile, please try on iOS Simulator, it should work 😄
